### PR TITLE
[Feature list] Introduce feature list

### PR DESCRIPTION
**Pull request by gauravchalana1**
Hi @gino-m ,

On click layer, the URL will change to '#fl'. PFA screenshots.

Screen Shot 2021-07-04 at 1 53 11 PM
Screen Shot 2021-07-04 at 1 56 34 PM

Please review. Thank you!

Contributes to #777.

### CODE REVIEW

File: web-ng/src/app/components/layer-list-item/layer-list-item.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 limitations under the License.
 -->
 <mat-list-item>
-  <h3 matLine>
+  <h3 matLine (click)="onLayerListSelect()">
```
Comment (gino-m) [inline web-ng/src/app/components/layer-list-item/layer-list-item.component.html:17]: Let's remove this until we have something to show in the list.

File: web-ng/src/app/components/feature-list/feature-list.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<p>Feature List</p>
```
Comment (gino-m) [inline web-ng/src/app/components/feature-list/feature-list.component.html:1]: Please add copyright headers here and throughout.

File: web-ng/src/app/components/feature-list/feature-list.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+<!--
+    * Todo: Display list of features for the layer!
```
Comment (gino-m) [inline web-ng/src/app/components/feature-list/feature-list.component.html:4]: Nit: add issue number like so:
```
<!--
  TODO(#...): ..
-->
```

File: web-ng/src/app/services/navigation/navigation.service.ts [hunk 4]
```diff
@@ -151,6 +155,12 @@ export class NavigationService {
     this.setFragmentParams(new HttpParams({ fromObject: newParam }));
   }
 
+  selectLayerFeature(id: string) {
```
Comment (gino-m) [inline web-ng/src/app/services/navigation/navigation.service.ts:158]: Maybe `showFeatureList(layerId: string)` would be clearer?

**Review by gino-m — CHANGES_REQUESTED** [review]
A few suggestions. Thanks for the change.

**Review by gino-m — APPROVED** [review]
LGTM!! Welcome back! 👏

**Comment by gauravchalana1** [comment]
Thank you Sir 🙂

**Pull request merged**